docs/installation/reference-installer.mdx

```diff
@@ -1,76 +1,40 @@
+type: reference
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/installation/reference-installer.mdx
+---
 
-import { Install, InstallBox } from '../../../src/components/Install/'
+import { Install, InstallBox } from '@site/src/components/Install'
 
-# Installation guide
+# Installation reference
 
-Netdata is a monitoring agent designed to run on all your systems: physical and virtual servers, containers, even
-IoT/edge devices. Netdata runs on Linux, FreeBSD, macOS, Kubernetes, Docker, and all their derivatives.
+Netdata offers many installation methods to support all your physical/virtual servers.
 
-The best way to install Netdata is with our [**automatic one-line installation
-script**](#automatic-one-line-installation-script), which works with all Linux distributions, or our [**.deb/rpm
-packages**](/packaging/installer/methods/packages.md), which seamlessly install with your distribution's package
-manager.
+The **recommended** way to install Netdata on a Linux node (physical, virtual, container, IoT) is our [one-line
+installer script (`kickstart.sh`)](/docs/get-started.mdx#install-on-linux-with-one-line-installer-recommended), which
+automatically installs dependencies and builds Netdata from its source code. For those who don't want to install Netdata
+directly, we also recommend [running Netdata with Docker](/docs/get-started.mdx#run-netdata-with-docker).
 
-If you want to install Netdata with Docker, on a Kubernetes cluster, or a different operating system, see [Have a
-different operating system, or want to try another
-method?](#have-a-different-operating-system-or-want-to-try-another-method)
+## Supported operating systems
 
-Some third parties, such as the packaging teams at various Linux distributions, distribute old, broken, or altered
-packages. We recommend you install Netdata using one of the methods listed below to guarantee you get the latest
-checksum-verified packages.
+We run automated checks on every PR to ensure that Netdata installs and runs properly on the following Linux
+distributions:
 
-Netdata collects anonymous usage information by default and sends it to our self hosted [PostHog](https://github.com/PostHog/posthog) installation. PostHog is an open source product analytics platform, you can read
-about the information collected, and learn how to-opt, on our [anonymous statistics](/docs/anonymous-statistics.md)
-page.
+- Ubuntu `21.04`, `20.10`, `20.04`, `18.04`, `16.04`
+- Debian `10`, `9`
+- Alpine `edge`, `3.13`, `3.12`, `3.11`, `3.10`
+- Arch Linux `latest`
+- Fedora `34`, `33`, `32`
+- CentOS `8`, `7`
+- OpenSUSE `tumbleweed:latest`, `leap:15.2`
+- Clear Linux `latest`
 
-The usage statistics are _vital_ for us, as we use them to discover bugs and prioritize new features. We thank you for
-_actively_ contributing to Netdata's future.
+Netdata runs on other distributions and operating systems as well, but they aren't officially supported by our team.
```

Comment:
This is a bold claim. We need to update the installation platform modals with a disclaimer (e.g macos).

Reply:
We’ve been discussing our formal support policy some recently. There’s a meeting about this scheduled for next Monday. Assuming we end up with a concrete and well defined support policy there, the plan is t then update `packaging/DISTRIBUTIONS.md` accordingly and make an effort to keep it up to date, in which case this whole section should probably just be a link to that page.

Reply:
@OdysLam I guess I'm not sure how this is a bold claim. Is it not true that Netdata runs on FreeBSD, Alpine, etc etc?

But @Ferroin, I'm looking forward to seeing what you come up with about the support policy and the `DISTRIBUTIONS.md` doc. I'll very happily streamline the structure of the docs around installation when there's more clarity there.